### PR TITLE
4754 Remove duplicate indexes in Land DB

### PR DESCRIPTION
**Pull request by alexGrandolph**
Trello card: [Remove duplicate indexes in Land DB](https://trello.com/c/FPfvq4aG/4754-remove-duplicate-indexes-in-land-db)

- green specs
- run `rails db:migate` to run migrations on your local machine
- I could use some feedback on how to test this, thank you!

Once this is merged, we can then pull the migrations into ADR for this card: 
[Pull in new land migrations for removing duplicate indexes](https://trello.com/c/TfPrHQO8/4702-pull-in-new-land-migrations-for-removing-duplicate-indexes)

### CODE REVIEW

File: db/migrate/20230526200145_remove_ad_type_index_on_attributions.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class RemoveAdTypeIndexOnAttributions < ActiveRecord::Migration[7.0]
+  def change
+    remove_index 'land.attribtions', column: :ad_type_id if index_exists?('land.attribtions', :ad_type_id)
```
Comment (stevenskim) [inline db/migrate/20230526200145_remove_ad_type_index_on_attributions.rb]: Typo attributions is missing u

**Review by stevenskim — CHANGES_REQUESTED** [review]
There is a typo, can you fix and reapply to update structure.sql? This change will require ADR pre to be updated as well.

**Review by stevenskim — APPROVED** [review]
👍

**Comment by bethanymuir** [comment]
[Remove duplicate indexes in Land DB](https://trello.com/c/FPfvq4aG/4754-remove-duplicate-indexes-in-land-db)

**Pull request merged**